Tests/UnitTests/Views/TableViewTests.cs

```diff
@@ -105,6 +101,8 @@ public void CellEventsBackgroundFill ()
             style.ColorGetter = e => { return scheme; };
         }
 
+        // Required or won't draw properly
+        Application.Driver.Clip = new Region (tv.Frame);
```

Comment:
Staying in the `UnitTests` project there is no problem. Only in the Parallelizable project that would be a problem.

Reply:
It is problem, if you remove it then test fails.  This commit I removed the Application.Top bit 

Alone removing top causes issues with repeated draws.  This line fixes that

Reply:
I think you didn't understand. What @tig said was "`Can the dependency on Application.Top be removed so this can move to a parallelizable test?`". If a test can't be run without the dependency the leave as is, otherwise remove the dependencies and move to a parallelizable test. By the fact it still need the `Application` it can't be removed, So, it can't, don't do anything 😃

Reply:
Ah OK, thanks for explaining,  I will see which can be moved and which cannot